staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2341,6 +2341,8 @@ const (
 	TerminationMessagePathDefault string = "/dev/termination-log"
 )
 
+type InitContainer = Container
```

Comment:
If you add a good comment here, it looks like at least some of the doc-gen tools DTRT - neat!